etc/listproviders/locations.properties

```diff
@@ -0,0 +1,6 @@
+list.name=locationFilter
```

Comment:
Thanks for factoring out the hard-coded lists, but as for this list, I don't think that it is of any use. Please remove the list "locationFilter" or provide a reason why you think it is needed.